composer.json

```diff
@@ -36,15 +73,9 @@
         "mikey179/vfsstream": "~1.2",
         "phpunit/phpunit": ">=4.8.28 <5",
         "symfony/css-selector": "~2.8",
-        "geerlingguy/drupal-vm": "^4.2",
-        "drupal/browsersync": "^1.0",
```

Comment:
All these deps were moved to [kalaconfig](https://github.com/kalamuna/kalaconfig).